scripts/build.sh

```diff
@@ -17,4 +39,4 @@ nix eval --extra-experimental-features nix-command --write-to "$FLAKE" --file "g
 nixfmt "$FLAKE"
 
 # Build the system configuration
-nixos-rebuild switch --show-trace --impure --flake .#"$(cat /etc/hostname)"
+nixos-rebuild $action --show-trace --impure --flake .#"$(cat /etc/hostname)" ${extraBuildArgs[*]}
```

Comment:
Apparently `"${extraBuildArgs[@]}"` is more correct here. Tested and works for me.
![image](https://github.com/user-attachments/assets/8dc05cf3-1163-49e8-ac5c-871760b2586c)

Reply:
I was not getting that warning. The problem with enclosing arguments in quotes is that they don't get spread. For example, if we want to pass the arguments `-a -b` and we surround the with quotes, like the linters wants, they will be passed as a single string value.

Should I use `xargs` to pipe the arguments, or find a way to disable linting on that line?